library/repo.go

```diff
@@ -31,6 +33,7 @@ type Repo struct {
 	AllowTag     *bool   `json:"allow_tag,omitempty"`
 	AllowComment *bool   `json:"allow_comment,omitempty"`
 	PipelineType *string `json:"pipeline_type,omitempty"`
+	PreviousName *string `json:"name_history,omitempty"`
```

Comment:
Should this be `previous_name`?

```suggestion
	PreviousName *string `json:"previous_name,omitempty"`
```